### ISSUE TITLE
Cannot reimplement tanet model in the model zoo using the provided configs on something v1 datasets

### ISSUE THREAD

**Issue by dy1998** [open-mmlab/mmaction2#1230]
**Describe the issue**

I download the tanet model from model zoo, and test it with provided configs on sth v1. However, it cannot get accurate as the table show. 

**Reproduction**

1. What command or script did you run?

```
python tools/test.py configs/recognition/tanet/tanet_r50_1x1x16_50e_sthv1_rgb.py   tanet_r50_1x1x16_50e_sthv1_rgb_20210630-7c19303c.pth --eval top_k_accuracy mean_class_accuracy     --out result.json
```

2. What config dir you run?

```
_base_ = [
    '../../_base_/models/tanet_r50.py', '../../_base_/default_runtime.py',
    '../../_base_/schedules/sgd_tsm_50e.py'
]

# model settings
model = dict(
    backbone=dict(num_segments=16),
    cls_head=dict(num_classes=174, num_segments=16, dropout_ratio=0.6))

# dataset settings
dataset_type = 'RawframeDataset'
data_root = '/home/amax/sthv1/20bn-something-something-v1/'
data_root_val = '/home/amax/sthv1/20bn-something-something-v1/'
ann_file_train = 'data/sthv1/sthv1_train_list_rawframes.txt'
ann_file_val = 'data/sthv1/sthv1_val_list_rawframes.txt'
ann_file_test = 'data/sthv1/sthv1_val_list_rawframes.txt'

sthv1_flip_label_map = {2: 4, 4: 2, 30: 41, 41: 30, 52: 66, 66: 52}
img_norm_cfg = dict(
    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_bgr=False)

train_pipeline = [
    dict(type='SampleFrames', clip_len=1, frame_interval=1, num_clips=16),
    dict(type='RawFrameDecode'),
    dict(type='Resize', scale=(-1, 256)),
    dict(
        type='MultiScaleCrop',
        input_size=224,
        scales=(1, 0.875, 0.75, 0.66),
        random_crop=False,
        max_wh_scale_gap=1,
        num_fixed_crops=13),
    dict(type='Resize', scale=(224, 224), keep_ratio=False),
    dict(type='Flip', flip_ratio=0.5, flip_label_map=sthv1_flip_label_map),
    dict(type='Normalize', **img_norm_cfg),
    dict(type='FormatShape', input_format='NCHW'),
    dict(type='Collect', keys=['imgs', 'label'], meta_keys=[]),
    dict(type='ToTensor', keys=['imgs', 'label'])
]
val_pipeline = [
    dict(
        type='SampleFrames',
        clip_len=1,
        frame_interval=1,
        num_clips=16,
        test_mode=True),
    dict(type='RawFrameDecode'),
    dict(type='Resize', scale=(-1, 256)),
    dict(type='CenterCrop', crop_size=224),
    dict(type='Normalize', **img_norm_cfg),
    dict(type='FormatShape', input_format='NCHW'),
    dict(type='Collect', keys=['imgs', 'label'], meta_keys=[]),
    dict(type='ToTensor', keys=['imgs'])
]
test_pipeline = [
    dict(
        type='SampleFrames',
        clip_len=1,
        frame_interval=1,
        num_clips=16,
        twice_sample=True,
        test_mode=True),
    dict(type='RawFrameDecode'),
    dict(type='Resize', scale=(-1, 256)),
    dict(type='ThreeCrop', crop_size=256),
    dict(type='Normalize', **img_norm_cfg),
    dict(type='FormatShape', input_format='NCHW'),
    dict(type='Collect', keys=['imgs', 'label'], meta_keys=[]),
    dict(type='ToTensor', keys=['imgs'])
]
data = dict(
    videos_per_gpu=4,
    workers_per_gpu=2,
    test_dataloader=dict(videos_per_gpu=1),
    train=dict(
        type=dataset_type,
        ann_file=ann_file_train,
        data_prefix=data_root,
        filename_tmpl='{:05}.jpg',
        pipeline=train_pipeline),
    val=dict(
        type=dataset_type,
        ann_file=ann_file_val,
        data_prefix=data_root_val,
        filename_tmpl='{:05}.jpg',
        pipeline=val_pipeline),
    test=dict(
        type=dataset_type,
        ann_file=ann_file_test,
        data_prefix=data_root_val,
        filename_tmpl='{:05}.jpg',
        pipeline=test_pipeline))
evaluation = dict(
    interval=1, metrics=['top_k_accuracy', 'mean_class_accuracy'])

# optimizer
optimizer = dict(lr=0.005, weight_decay=0.001)
lr_config = dict(policy='step', step=[30, 40, 45])

# runtime settings
work_dir = './work_dirs/tanet_r50_1x1x16_50e_sthv1_rgb/'

```

3. Did you make any modifications on the code or config? Did you understand what you have modified?
A: I only change the path of dataset.

4. What dataset did you use?
A:something-something v1

**Environment**

fatal: Not a git repository (or any parent up to mount point /data)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
sys.platform: linux
Python: 3.7.11 (default, Jul 27 2021, 14:32:16) [GCC 7.5.0]
CUDA available: True
GPU 0,1: TITAN Xp
CUDA_HOME: /usr/local/cuda-10.2
NVCC: Cuda compilation tools, release 10.2, V10.2.89
GCC: gcc (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609
PyTorch: 1.7.1
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v1.6.0 (Git Hash 5ef631a030a6f73131c77892041042805a06064f)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 10.2
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75
  - CuDNN 7.6.5
  - Magma 2.5.2
  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_VULKAN_WRAPPER -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, 

TorchVision: 0.8.2
OpenCV: 4.5.3
MMCV: 1.3.15
MMCV Compiler: GCC 5.4
MMCV CUDA Compiler: 10.2
MMAction2: 0.19.0+

**Results**

If applicable, paste the related results here, e.g., what you expect and what you get.

```
python tools/test.py configs/recognition/tanet/tanet_r50_1x1x16_50e_sthv1_rgb.py   tanet_r50_1x1x16_50e_sthv1_rgb_20210630-7c19303c.pth --eval top_k_accuracy mean_class_accuracy     --out result.json
Use load_from_local loader
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 11522/11522, 4.6 task/s, elapsed: 2523s, ETA:     0s
writing results to result.json

Evaluating top_k_accuracy ...

top1_acc        0.5030
top5_acc        0.7814

Evaluating mean_class_accuracy ...

mean_acc        0.4623
top1_acc: 0.5030
top5_acc: 0.7814
mean_class_accuracy: 0.4623
```

**Comment by dreamerlin** [comment]
Try another model like 'tsm'? Actually, we just take the Regression Testing recently and show no difference. Besides, the ACC result shown in the log is the same as the test one.

**Comment by kennymckormick** [comment]
Hi, @dy1998. The problem is fixed in PR #1306, thanks @dreamerlin.

**Comment by LEM0NTE** [comment]
> > 好的，非常感谢您解决这个问题。我们将尝试找出此问题的原因。现在，您能否提出一个 PR 来帮助我们更新 TANet 的这些数字？
> 
> 我只做了 sth-v1 数据集的实验。此外，我已经在我以前的conda env（torch1.7.1）上测试了TSM的官方代码，并且也得到了衰减。但是，当我在 torch1.5 env 上测试它时，它的工作原理如官方日志所示。所以我认为火炬的版本可能有问题。
> 
> 顺便说一句，我唯一改变的是sth_v1数据集的data_root。很抱歉我不知道如何提出 PR，因为我没有提出 PR。

您好，请问您是否方便分享您下载到的sth v1数据集嘛。它的官网已经503了，或者您可以分享您的下载方法嘛，十分感谢
